atomic_reactor/inner.py

```diff
@@ -857,21 +855,6 @@ def build_docker_image(self) -> BuildResult:
                 logger.error("one or more prepublish plugins failed: %s", ex)
                 raise
 
-            if self.data.build_result.is_image_available():
```

Comment:
I removed this to avoid any changes here, as this will be removed anyway in https://github.com/containerbuildsystem/atomic-reactor/pull/1810/. (I will rebase this after that is merged)